src/types/models/listing.ts

```diff
@@ -113,6 +113,7 @@ export interface Listing
   typeSlug: string
   active: boolean
   bodyColorGroup: string
```

Comment:
Do we still need it?

Reply:
I think not, but I left it just in case some project is still using bodyColorGroup prop.
We can remove it later.

Reply:
removing is already part of the task, please check if you still need it, and if not, let's remove it and release a breaking change

Reply:
plus one. backend will likely remove it on their end anyway